dev-python/pymad/pymad-0.10.ebuild

```diff
@@ -0,0 +1,27 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python2_7 python3_{5,6,7} )
```

Comment:
remove py2 while at it, so we don't have to revisit it

Reply:
done